packages/ibm-products/src/components/Tearsheet/Tearsheet.tsx

```diff
@@ -59,7 +59,7 @@ export interface TearsheetProps extends PropsWithChildren {
    *
    * See https://react.carbondesignsystem.com/?path=/docs/components-button--default#component-api
    */
-  actions?: ButtonProps<'button'>[];
+  actions: ButtonProps<React.ElementType>[];
```

Comment:
`actions` shouldn't be required for tearsheets. Just wondering about this change